content/docs/notebooks/setup.md

```diff
@@ -86,34 +86,38 @@ central dashboard](/docs/components/central-dash/overview/).
   ensures that the new notebook server is in a namespace that you can access.
 
 1. <a id="docker-image"></a> Select a Docker **image** for the baseline 
-  deployment of your notebook server. You can choose from a range of *standard* 
-  images or specify a *custom* image:
+  deployment of your notebook server. You can specify a custom image or
+  choose from a range of standard images:
 
-  * **Standard**: The standard Docker images include typical machine learning 
+  * **Custom image**: If you select the custom option, you must specify a Docker
+    image in  the form `registry/image:tag`. For guidelines on creating a Docker
+    image for your notebook, see the guide to 
+    [creating a custom Jupyter image](/docs/notebooks/custom-notebook/).
+
+  * **Standard image**: Click the **Image** dropdown menu to see the list of
+    available images. The standard Docker images include typical machine learning 
     (ML) packages that you can use within your Jupyter notebooks on 
-    this notebook server. Select an image from the **Image** dropdown menu.
+    this notebook server. Click one of the images to select it.
+    
     The image names indicate the following choices:
 
-      * A TensorFlow version (for example, `tensorflow-1.13.1`). Kubeflow offers
-        a CPU and a GPU image for each minor version of TensorFlow.
+      * A TensorFlow version (for example, `tensorflow-1.15.2`).
       * `cpu` or `gpu`, depending on whether you want to train your model on a CPU 
         or a GPU. 
         
           * If you choose a GPU image, make sure that you have GPUs 
             available in your Kubeflow cluster. Run the following command to check 
             if there are any GPUs available:
-            `kubectl get nodes "-o=custom-columns=NAME:.metadata.name,GPU:.status.allocatable.nvidia\.com/gpu"`
+            ```
+            kubectl get nodes "-o=custom-columns=NAME:.metadata.name,GPU:.status.allocatable.nvidia\.com/gpu"
+            ```
           * If you have GPUs available, you can schedule your server on a GPU node 
-            in the **Extra Resources** section at the bottom of the form. For 
-            example, to reserve two GPUs, enter the following JSON code: 
-          `{"nvidia.com/gpu": 2}`
-      * Kubeflow version (for example, `v0.5.0`).
-    
+            in the **GPUs** section at the bottom of the form.
```

Comment:
Would it make more sense to rephrase the `you can schedule your server on a GPU node in the GPUs section at the bottom of the form` to something like:

`you can schedule your server on a GPU node by specifying a number of GPUs to be attached to the server under the **GPUs** section at the bottom`